package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@catena-x/portal-shared-components",
-  "version": "2.1.11",
+  "version": "2.1.14",
```

Comment:
why not 2.1.12?

Reply:
Made the change. And yes I ran the linter command already.